modules/brainbrowser/js/brainbrowser.loris.js

```diff
@@ -748,17 +749,52 @@ $(function() {
         minc_ids_arr = [minc_ids];
     }
 
+    var request;
     for (i = 0; i < minc_ids_arr.length; i += 1) {
 
-        minc_volumes.push({
-            type: 'minc',
-	    header_url: "",
-	    raw_data_url: loris.BaseURL + "/brainbrowser/ajax/minc.php?minc_id=" + minc_ids_arr[i],
-            template: {
+      request = $.ajax({
```

Comment:
perhaps it's outside the scope of this PR but i think it would be a good idea to condense this into just 1 ajax call if possible
other than that, LGTM